docker/bin/e2e/sqlite/e2e-env-reset.sh

```diff
@@ -5,22 +5,20 @@
 docker compose down
 
 rm -f ./storage/database/torrust_index_backend_e2e_testing.db
-rm -f ./storage/database/torrust_tracker_e2e_testing.db
+rm -f ./storage/tracker/lib/database/torrust_tracker_e2e_testing.db
 
 # Generate storage directory if it does not exist
 mkdir -p "./storage/database"
 
 # Generate the sqlite database for the index backend if it does not exist
 if ! [ -f "./storage/database/torrust_index_backend_e2e_testing.db" ]; then
     # todo: it should get the path from config.toml and only do it when we use sqlite
```

Comment:
This todo can be removed.

Reply:
Hi @da2ce7  I meant that there should be only one place to define the database. I think the `config-idx-back.sqlite.local.toml` or the `config-idx-back.sqlite.local.toml` files` is the original place where that info is located. At least now we have another env var
`TORRUST_IDX_BACK_CONFIG` which is better than the hardcoded value. Anyway, I think having to places to define the same value can lead to confusion:

- TORRUST_IDX_BACK_CONFIG
- TORRUST_IDX_BACK_MYSQL_DATABASE

Having the second value overwriting the first one is something that the user might expect, but I think we have to find an explicit convention for this behaviour. It's something to consider in the [configuration overhaul](https://github.com/torrust/torrust-tracker/issues/401).

For now, I will remove the todo and I will use the env var TORRUST_IDX_BACK_MYSQL_DATABASE in all the scripts.